accord-core/src/test/java/accord/impl/PrefixedIntHashKey.java

```diff
@@ -310,6 +310,10 @@ public int compareTo(@Nonnull RoutableKey that)
                     int rightInt = that instanceof Sentinel ? (((Sentinel) that).isMin ? -1 : 1) : 0;
                     rc = Integer.compare(leftInt, rightInt);
                 }
+                else if (!(this instanceof PrefixedIntRoutingKey) && (!(that instanceof PrefixedIntRoutingKey)))
```

Comment:
`PrefixedIntRoutingKey -> Hash | Sentinel`
`PrefixedIntHashKey -> Hash | Sentinel | Key`

so this is just saying `else if (this.isKey() && that.isKey())`?  Since you are checking the key it might be best to do that?

Reply:
I think the key class might be a common super class so we don’t know if it is a key, only if it isn’t? Will double check

Reply:
`Key` isn't a common super class, it's the final top level class.  We have 3 top level types: `Key | Hash | Sentinel`